src/main/java/com/example/springBoot_hibernate_ferrocarriles/service/ItinerarioService.java

```diff
@@ -0,0 +1,45 @@
+package com.example.springBoot_hibernate_ferrocarriles.service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.example.springBoot_hibernate_ferrocarriles.model.Itinerario;
+import com.example.springBoot_hibernate_ferrocarriles.repository.ItinerarioRepository;
+
+@Service
+public class ItinerarioService {
+	@Autowired
```

Comment:
The latest versions of Spring don't require `@Atowired` annotation anymore. try if it is your case.